tests/components/homewizard/conftest.py

```diff
@@ -57,6 +59,11 @@ def mock_homewizardenergy():
                 json.loads(load_fixture("homewizard/state.json"))
             )
         )
+        client.system = AsyncMock(
+            return_value=System.from_dict(
```

Comment:
This means the JSON is loaded every time for every test that uses this fixture. That’s inefficient. Use a side effect with a lambda that loads it on demand.

```suggestion
            side_effect=lambda: System.from_dict(
```